src/main/java/org/broadinstitute/dsde/consent/ontology/datause/builder/ConsentRestrictionBuilder.java

```diff
@@ -86,4 +126,15 @@ else if (dataUse.getGender().equalsIgnoreCase("female")) {
         return restriction;
     }
 
+    private Boolean isMethodsResearchProhibited(DataUse dataUse) {
+        if (isPresent(dataUse.getMethodsResearch()))
```

Comment:
just want to double check that an empty value should default to false here, but above (line 108) an empty value defaults to true.

Reply:
Good question, and yes, I needed to triple check just to be sure this is correct. This is the code in ORSP that sets the value:
```
if (dataUseRestriction.methodsResearchExcluded) {
     dataUseDTO.methodsResearch = true
}
```
So ... 
* `methodsResearch:true` means that it is there is a restriction.
* `methodsResearch:false` means that it is there is no restriction.